pages/index.tsx

```diff
@@ -50,9 +50,9 @@ export const Index: NextPage = () => {
                 Connect; Learn; Build;
               </Heading>
               <Text fontSize="1em" color="gray.700" my="5">
-                We are a community of student developers, designers, and
-                entrepreneurs who are passionate about building the future of
-                the software.
+                We are a community of student developers, designers, and entrepreneurs
```

Comment:
The other description I've seen used for VikeLabs is, "VikeLabs is a collective of students who learn to build, deploy and test software apps quickly. The group views UVic as a kind of laboratory for testing solutions to problems that exist within the campus community." 

@liamshatzel do you have a preference for which we use on the website? Or we could try a combo of that one and the one currently on the website?

Also if we keep the change currently in the PR, could we please end the description with a period?

Reply:
An option could be, "VikeLabs is a collective of students who learn to build, deploy, and test software apps. We are a community of student developers, designers, and entrepreneurs who are passionate about designing software solutions for students and the UVic campus community."

Reply:
Yeah lets use that one that Amanda wrote out

Reply:
I like the second one! 

_VikeLabs is a collective of students who learn to build, deploy, and test software apps. We are a community of student developers, designers, and entrepreneurs who are passionate about designing software solutions for students and the UVic campus community_